crates/epaint/src/tessellator.rs

```diff
@@ -757,12 +819,27 @@ fn cw_signed_area(path: &[PathPoint]) -> f64 {
     }
 }
 
+/// See [`TessellationOptions::max_feathering_normal_length_sq`].
+fn clamp_normal_for_feathering(normal: Vec2, max_feathering_normal_length_sq: f32) -> Vec2 {
+    if normal.length_sq() > max_feathering_normal_length_sq {
+        normal * (max_feathering_normal_length_sq / normal.length_sq())
```

Comment:
I think this is incorrect.

if `normal = [10, 0]` for instance, and max_feathering_normal_length = 2, so max_feathering_normal_length_sq = 4, then we would divide by 100/4 instead of 10/2.

```suggestion
    if normal.length_sq() > sqr(max_feathering_normal_length) {
        max_feathering_normal_length * normal.normalize()
```